src/test/java/uk/gov/hmcts/reform/logging/layout/ReformLoggingLayoutTest.java

```diff
@@ -0,0 +1,154 @@
+package uk.gov.hmcts.reform.logging.layout;
+
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.joran.JoranConfigurator;
+import ch.qos.logback.core.joran.spi.JoranException;
+import org.junit.After;
+import org.junit.experimental.theories.DataPoints;
+import org.junit.experimental.theories.Theories;
+import org.junit.experimental.theories.Theory;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.reform.logging.exception.AbstractLoggingException;
+import uk.gov.hmcts.reform.logging.exception.AlertLevel;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InvalidClassException;
+import java.io.PrintStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(Theories.class)
+public class ReformLoggingLayoutTest {
+
+    private PrintStream old = System.out;
+    private ByteArrayOutputStream baos = null;
+    private final Logger log = LoggerFactory.getLogger(ReformLoggingLayoutTest.class);
+
+    private enum LogbackConfig {
+        C1("logback-test-enable-thread.xml"),
+        C2("logback-test-custom-date-format.xml"),
+        C3("logback-test-disable-alert-level.xml");
+
+        private String value;
+
+        private LogbackConfig(String configResource) {
+            value = configResource;
+        }
+    }
+
+    @DataPoints
+    public static LogbackConfig[] configValues = LogbackConfig.values();
```

Comment:
test methods take no parameters, is this needed? (both datapoints and enum)

Reply:
Darn. Probably overlooked after all the tests passed. One the other hand, once I've written these kind of tests I realised (too late) `Theory` is completely not needed. Just set up as it is now but don't use fancy `Theories` runner.
Let me delete more lines